### PR TITLE
NMS-8216, NMS-2504, HZN-498, HZN-704, HZN-588: Discovery admin page for ad hoc scans + bugfixes

### PR DESCRIPTION
**Pull request by soleger**
This PR contains the following enhancements:
- Fixes bug introduced during recent Discovery UI work (duplicated retries field)
- Adds "chunksize" config parameter to UI
- Cleans up look-and-feel of top config form on discovery admin page
- Duplicates the admin page to provide a new admin page for ad hoc, one-time Discovery scans
- Ensures that default values are always filled in where appropriate for timeouts and retries
- Provides a `<select>` dropdown for Location and Foreign Source names

JIRA: 
- http://issues.opennms.org/browse/NMS-8216
- http://issues.opennms.org/browse/NMS-2504
- http://issues.opennms.org/browse/HZN-498
- http://issues.opennms.org/browse/HZN-704
- http://issues.opennms.org/browse/HZN-588

Bamboo: 
- http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS595

### CODE REVIEW

File: features/discovery/src/main/java/org/opennms/netmgt/discovery/DiscoveryTaskExecutorCamelImpl.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+	@Produce(property="endpointUri")
+	DiscoveryTaskExecutor m_proxy;
+
+	public DiscoveryTaskExecutorCamelImpl(final String endpointUri) {
+		super(endpointUri);
+	}
+
+	/**
+	 * Send the incoming {@link SyslogConnection} message into the Camel route
```
Comment (j-white) [inline features/discovery/src/main/java/org/opennms/netmgt/discovery/DiscoveryTaskExecutorCamelImpl.java]: syslog?

**Comment by j-white** [comment]
Using the run scan page, I run a scan against a new location (with no Minion attached) I'm presented with an exception:
```
org.apache.camel.ExchangeTimedOutException
	at org.apache.camel.component.jms.reply.ReplyManagerSupport.processReply(ReplyManagerSupport.java:133)
	at org.apache.camel.component.jms.reply.TemporaryQueueReplyHandler.onTimeout(TemporaryQueueReplyHandler.java:61)
```

File: opennms-webapp/src/main/webapp/admin/discovery/edit-scan.jsp [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+<%
+	response.setDateHeader("Expires", 0);
+	response.setHeader("Pragma", "no-cache");
+	if (request.getProtocol().equals("HTTP/1.1")) {
+		response.setHeader("Cache-Control", "no-cache");
+	}
+%>
+<% String breadcrumb1 = "<a href='admin/index.jsp'> Admin </a>"; %>
+<% String breadcrumb2 = "<a href='admin/discovery/index.jsp'> Discovery </a>"; %>
+<% String breadcrumb3 = "Modify Configuration"; %>
```
Comment (j-white) [inline opennms-webapp/src/main/webapp/admin/discovery/edit-scan.jsp]: We probably want to use a different bread crumb here

**Comment by soleger** [comment]
The "exchange timed out" problem is two issues:
- Timeouts not calculated correctly: http://issues.opennms.org/browse/HZN-628
- New issue to make the operation async: http://issues.opennms.org/browse/HZN-704

**Pull request merged**